src/core.js

```diff
@@ -37,9 +37,11 @@ function stage(cwd, {alias}) {
       aliasProc.on('data', (error) => reject(new Error(error)));
       aliasProc.on('close', (code) => {
         log.info(`> Alias ready ${alias}`);
-        resolve(alias);
       });
     });
+    nowProc.stdout.on('close', () => {
+      resolve(alias);
```

Comment:
Does this have the potential to return an alias that might not be set if `aliasProc` fails for some reason?

Maybe we should wait to start the `aliasProc` until after `nowProc` closes. Could give us more certainty about the chain of events. `deploy -> alias` vs `start deploy -> start alias -> ??`